mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergTestUtils.java

```diff
@@ -265,4 +278,57 @@ public static void validateFiles(Table table, Configuration conf, JobID jobId, i
     Assert.assertEquals(dataFileNum, dataFiles.size());
     Assert.assertFalse(new File(HiveIcebergOutputCommitter.generateJobLocation(conf, jobId)).exists());
   }
+
+  /**
+   * Lazy implementation for checking the the returned results from a SELECT statement are the same than the inserted
```

Comment:
Why is it a lazy implementation?

Reply:
No ordering is implemented which should be done in a fool-proof implementation.
Expects the records in a given order

Reply:
Ah I see. I think I misunderstood it as something to do with lazy loading/initialization...

Reply:
Updated the comment, could you please check?
Thanks,
Peter

Reply:
Looks good, thanks